src/i18n/messages/de.messages.json

```diff
@@ -0,0 +1,123 @@
+{
+  "base": {
+    "searchBox": {
+      "clear": "Dégager",
+      "placeholder": "Que cherchez-vous?"
+    },
+    "popularSearches": {
+      "title": "RECHERCHES POPULAIRES"
+    },
+    "historyQueries": {
+      "clear": "Tout effacer",
+      "removeLabel": "Supprimer {suggestion} des requêtes d'historique",
+      "title": "RECHERCHES D'HISTORIQUE"
+    },
+    "myHistory": {
+      "title": "Mon histoire",
+      "subtitle": "Ceci est la liste complète des requêtes d'historique",
+      "message": "<span>Ces données sont stockées localement. Juste dans votre appareil et ne voyagez jamais en dehors de celui-ci.</span><span>Vous pouvez désactiver les requêtes d'historique quand vous le souhaitez.</span><span class=\"x-font-color--neutral-10 x-font-weight--bold\">Votre vie privée sous votre contrôle.</span>",
+      "noHistory": "Aucune activité d'historique à afficher",
+      "openButton": "Mon histoire",
+      "switch": {
+        "title": "Requêtes d'historique",
+        "enable": "Activer",
+        "disable": "Désactiver et effacer"
+      },
+      "confirmDisableModal": {
+        "title": "Désactiver les requêtes d'historique ?",
+        "message": "En désactivant les requêtes d'historique, vous effacez également toute la liste d'historique.",
+        "dismiss": "Annuler",
+        "confirm": "Désactiver"
+      }
+    },
+    "nextQueries": {
+      "title": "ET APRÈS"
+    },
+    "nextQueriesGroup": {
+      "message": "Ceux qui cherchaient <span class=\"x-text x-text--bold x-font-size--05\">\"{query}\"</span> également consulté:"
+    },
+    "nextQueryPreview": {
+      "message": "Ceux qui ont recherché {query} ont également consulté:",
+      "query": "{query}",
+      "totalResults": "({totalResults})",
+      "viewResults": "Voir tous les résultats"
+    },
+    "identifierResults": {
+      "title": "Recherche d'UGS"
```

Comment:
Could you use a translation for `identifier` instead of the acronym? We are trying to not use acronyms as there are many different ones for this purpose.

Reply:
Alright, surely,